### ISSUE TITLE
onnxruntime uses gpu

### ISSUE THREAD

**Issue by juntaosun** [mat490/Godot-ONNX-AI-Models-Loaders#2]
I noticed that your code loads the onnx model and uses the CPU by default.
You can add the following code to support gpu loading model and inference, which will be much faster than cpu.

Use GPU：
```
OrtSessionOptionsAppendExecutionProvider_CUDA(session_options, 0);
session = new Ort::Session(env, model_path_w.c_str(), session_options);
```

Hope you can add some examples of doing onnx model inference in Godot, thanks ！

**Comment by mat490** [comment]
You're right. I thank you very much for your contribution, I have been busy but these days I will work to implement it correctly. And I will try to make some examples of doing onnx model inference in Godot. 😁
